atc/worker/placement.go

```diff
@@ -49,26 +123,24 @@ func (strategy *VolumeLocalityPlacementStrategy) Choose(logger lager.Logger, wor
 		}
 	}
 
-	highestLocalityWorkers := workersByCount[highestCount]
-
-	return highestLocalityWorkers[strategy.rand.Intn(len(highestLocalityWorkers))], nil
+	return workersByCount[highestCount], nil
 }
 
-func (strategy *VolumeLocalityPlacementStrategy) ModifiesActiveTasks() bool {
+func (strategy *VolumeLocalityPlacementStrategyNode) ModifiesActiveTasks() bool {
 	return false
 }
 
-type FewestBuildContainersPlacementStrategy struct {
+type FewestBuildContainersPlacementStrategyNode struct {
 	rand *rand.Rand
```

Comment:
Looks like we can get rid of `rand` here, as well as in `LimitActiveTasksPlacementStrategyNode` and possibly `RandomPlacementStrategyNode` (see my comment there)

Reply:
Good catch.